gui/utils/compat.py

```diff
@@ -1,262 +0,0 @@
-# Backport of OrderedDict() class that runs on Python 2.4, 2.5, 2.6, 2.7 and pypy.
```

Comment:
Honestly we may be able to get rid of this crap altogether. This is a class that is used to mimic OrderedDict support for running in py2.6 and below. Since we only support 2.7 now, we might be able to nuke this.